source/includes/fact-retryable-writes-failover-election.rst

```diff
@@ -0,0 +1,10 @@
+Your application connection logic should include tolerance for 
```

Comment:
Why do we have two includes that say the same thing?  Doesn't it defeat the purpose of single sourcing?